bedrock/settings/base.py

```diff
@@ -227,10 +232,24 @@ def lazy_langs():
 LANGUAGES = lazy(lazy_langs, dict)()
 
 FEED_CACHE = 3900
-DOTLANG_CACHE = 600
+DOTLANG_CACHE = config('L10N_CACHE_TIMEOUT', default='600', parser=int)
```

Comment:
I had to set the value for `L10N_CACHE_TIMEOUT` to `10` locally when converting pages to avoid caching issues. I'm guessing others may also have similar issues, so documenting this may be useful?

Reply:
I can also have a much shorter default when `DEBUG == True`